src/components/ContentHighlights/ContentHighlightsDashboard.jsx

```diff
@@ -25,6 +27,16 @@ const ContentHighlightsDashboard = () => {
   const highlightSets = enterpriseCuration?.highlightSets;
   const [activeTab, setActiveTab] = useState(TAB_TITLES.highlights);
   const [isHighlightSetCreated, setIsHighlightSetCreated] = useState(false);
+  const sendTrackEvent = (tab) => {
```

Comment:
nit: `sendTrackEventTabSwitch` or similar to make this more explicit... e.g., say there's another Segment event we want to dispatch in this same component; which event is `sendTrackEvent` referring to?